system_tests/test_cli_package.py

```diff
@@ -241,54 +241,41 @@ def bootstrap_manager(self):
         if self.is_install_plugins():
             install_plugins = '--install-plugins'
         out = self._execute_command(
-            'bootstrap -p {0} -i {1} {2}'.format(
-                self.test_manager_blueprint_path,
-                self.remote_bootstrap_inputs_path,
+            """bootstrap -p {0} -i "{1}" {2}""".format(
+                self.manager_blueprint_path,
+                json.dumps(inputs).replace('"', "'"),
                 install_plugins),
             within_cfy_env=True)
 
-        self.assertIn('bootstrapping complete', out,
-                      'Bootstrap has failed')
+        self.assertIn('bootstrapping complete', out, 'Bootstrap has failed')
 
         self.manager_ip = self._manager_ip()
         self.client = CloudifyClient(self.manager_ip)
         self.addCleanup(self.teardown_manager)
 
-    def get_hello_world_url(self):
-        return EXAMPLE_URL.format(HELLO_WORLD_EXAMPLE_NAME, self.branch)
+    def get_example(self, example_url):
+        return example_url
 
-    def publish_hello_world_blueprint(self):
-        hello_world_url = self.get_hello_world_url()
+    def publish_hello_world_blueprint(self, source_archive):
         blueprint_id = 'blueprint-{0}'.format(uuid.uuid4())
         self.logger.info('Publishing hello-world example from: {0} [{1}]'
-                         .format(hello_world_url, blueprint_id))
+                         .format(source_archive, blueprint_id))
         self._execute_command('blueprints publish-archive '
                               '-l {0} -n {1} -b {2}'
-                              .format(hello_world_url,
+                              .format(source_archive,
                                       self.app_blueprint_file,
                                       blueprint_id),
                               within_cfy_env=True)
         return blueprint_id
 
-    def prepare_deployment(self):
-        self.local_deployment_inputs_path = \
-            self.cfy._get_inputs_in_temp_file(self.deployment_inputs,
-                                              self._testMethodName)
-        self.remote_deployment_inputs_path = \
-            os.path.join(self.client_cfy_work_dir,
-                         'deployment_inputs.json')
-        fab.put(self.local_deployment_inputs_path,
-                self.remote_deployment_inputs_path, use_sudo=True)
-
     def create_deployment(self, blueprint_id):
         deployment_id = 'deployment-{0}'.format(uuid.uuid4())
-        self.prepare_deployment()
 
         self.logger.info('Creating deployment: {0}'.format(deployment_id))
         self._execute_command(
-            'deployments create -b {0} -d {1} -i {2}'
+            """deployments create -b {0} -d {1} -i "{2}" """
             .format(blueprint_id, deployment_id,
-                    self.remote_deployment_inputs_path),
+                    json.dumps(self.deployment_inputs).replace('"', "'")),
```

Comment:
AFAIK the json.dumps string is different from our required inputs format of "key1=value1;key2=value2"